### PR TITLE
not recognizing the frame as a class, not sure why

### PR DESCRIPTION
**Pull request by sarastavnitser**

### CODE REVIEW

File: src/main/java/CurrencyExchangeFrame.java [hunk 1]
```diff
@@ -31,9 +35,12 @@ public class CurrencyExchangeFrame extends JFrame implements ItemListener {
     private String[] symbolsArray;
     private String[] descriptionsArray;
 
-    public CurrencyExchangeFrame() {
+    @Inject
+    public CurrencyExchangeFrame(CurrencyExchangePresenter presenter) {
```
Comment (andrewoid) [inline src/main/java/CurrencyExchangeFrame.java:41]: Put this and the Presenter in a package.

We shouldn't be using the default package.

File: src/test/java/CurrencyExchangePresenterTest.java [hunk 2]
```diff
@@ -1,17 +1,17 @@
+import currencyExchange.CurrencyExchangeFrame;
```
Comment (andrewoid) [inline src/test/java/CurrencyExchangePresenterTest.java:1]: Your tests should match the package names of the classes they are testing.

**Review by andrewoid — APPROVED** [review]

**Comment by andrewoid** [comment]
Removed the approval because I didn't realize your tests didn't pass.

**Comment by sarastavnitser** [comment]
So sorry, I thought we didn't have to work on the tests, will fix that now

**Review by andrewoid — APPROVED** [review]

**Pull request merged**